### ISSUE TITLE
catalog persistence does not work with symbiosis mode

### ISSUE THREAD

**Issue by justinj** [MaterializeInc/materialize#2755]
It's necessary to blow away mzdata each time you restart materialize when running in symbiosis mode, I'm not sure what needs to happen for this to work—I assume it's nontrivial.

**Comment by benesch** [comment]
Yeah, I think we might just want to disable persistence when running in
symbiosis maybe.
